homeassistant/components/climate/ecobee.py

```diff
@@ -190,7 +190,7 @@ def fan(self):
         """Return the current fan state."""
         if 'fan' in self.thermostat['equipmentStatus']:
             return STATE_ON
-        return STATE_OFF
```

Comment:
Is equipmentStatus what is _currently_ running, rather than what is configured? If that's the case, then I think it should stay `STATE_OFF`.

Reply:
@technicalpickles you're correct, this is the currently running status so I'll change this back to `STATE_OFF`, however it looks like the method name for this is incorrect and that's why we see `null` for `fan_mode` in the attributes.  I'm fixing that too.  should be pushed soon.